docs/plugins.md

```diff
@@ -57,6 +57,7 @@ To turn off plugin autoloading, use `--no-plugin-search` when using Prettier CLI
 
 ## Community Plugins
 
+- [`prettier-plugin-astro`](https://github.com/withastro/prettier-plugin-astro) by [**@withastro**](https://github.com/withastro)
```

Comment:
The plugin names are alphabetical sorted, you might want move to the second one.

Reply:
done, thx